### PR TITLE
Add support for R_AARCH64_MOVW_UABS_G* relocations

### PR DESCRIPTION
**Pull request by resrever**

### CODE REVIEW

**Review by galenbwill — COMMENTED** [review]
Can you provide any sample binaries I can use to test these relocations?

File: arch_arm64.cpp [hunk 1]
```diff
@@ -2951,6 +2952,16 @@ struct LDST_REG_UNSIGNED_IMM{
...
+struct MOV_WIDE_IMM{
+    uint32_t Rd:5;
+    uint32_t imm:16;
+    uint32_t shift:2;
+    uint32_t opcode:6;
+    uint32_t variant:3;
+};
+
+
+
```
Comment (galenbwill) [inline arch_arm64.cpp:2964]: Please remove the extra blank lines.

File: arch_arm64.cpp [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (galenbwill) [inline arch_arm64.cpp:1]: Please remove the extra blank line.

**Comment by resrever** [comment]
[lcd.zip](https://github.com/Vector35/arch-arm64/files/8575533/lcd.zip)
Here's a sample binary and I removed the extra blank line.

**Comment by lwerdna** [comment]
Thanks! I squashed your commits to 54383ba then rebased onto master.

**Pull request closed without merging**